.gitlab-ci.yml

```diff
@@ -0,0 +1,35 @@
+stages:
+  - validation
+  - build
+
+json:
+  image: debian:bullseye-slim
+  stage: validation
+  script:
+    - apt-get -q update
+    - apt-get -qy install yamllint
+    - yamllint info.puredata.Pd-extended.yaml
+
+flatpak:
```

Comment:
This is not needed, PRs get automatically built and test builds published for 5 days by our buildbot.

Reply:
Just noticed this is gitlab-ci.yml, so not even GitHub Actions. Do you plan to co-host that repo elsewhere? If so, please disregard my comment.

Reply:
I use GitLab CI for everything, I don't know GitHub Actions.  I have a dev fork for that here https://gitlab.com/pd-extended/flathub